variables.tf

```diff
@@ -77,7 +77,7 @@ variable "autoscale_min" {
 }
 
 variable "autoscale_max" {
-  default = "1"
+  default = "2"
```

Comment:
Is this safe? My concern would be to have more than one jenkins master online sharing the same FS at any given time.

Reply:
Was my concern as well.

`eb_environment` did not work when `autoscale_max` was set to `1` (Exception thrown by `AWS`).

> MinInstancesInService: Invalid option value: '1.0' (Namespace: 'aws:autoscaling:updatepolicy:rollingupdate', OptionName: 'MinInstancesInService'): The minimum instances in service for rolling updates (1) should be less than the maximum instance count for the Autoscaling group (1). 

Looks like we need to check the two values (`autoscale_min` and `autoscale_max`) and if `autoscale_min` >= `autoscale_max`, disable rolling updates.

Reply:
Actually, we don't need to change anything in `eb_environment` module.
Setting `updating_min_in_service` to `0` and `autoscale_max` to `1` worked.